draft-ietf-quic-recovery.md

```diff
@@ -1007,8 +1007,8 @@ OnAckReceived(ack, pn_space):
     largest_acked_packet[pn_space] =
         max(largest_acked_packet[pn_space], ack.largest_acked)
 
-  // Nothing to do if there are no newly acked packets.
   newly_acked_packets = DetermineNewlyAckedPackets(ack, pn_space)
+  // Nothing to do if there are no newly acked packets.
```

Comment:
This change looks like a regression, or it is as at least unrelated: the intent before was to comment about the call to `DetermineNewlyAckedPackets()` and the use of the returned value.  I think that's still the right way to frame this.